src/main/modules/oidc/index.ts

```diff
@@ -37,6 +37,7 @@ export class OidcMiddleware {
 
       req.session.user = response.data;
       req.session.user.jwt = jwt_decode(response.data.id_token);
+      req.session.user.roles = req.session.user.jwt.roles;
```

Comment:
maybe it would be better to set the isSuperAdmin property here instead of doing this.